package.json

```diff
@@ -33,16 +33,17 @@
     }
   },
   "dependencies": {
-    "agentkeepalive": "^4.1.3",
+    "@open-cluster-management/security-middleware": "github:open-cluster-management/security-middleware",
```

Comment:
can we remove the reference below now?

Reply:
Do you mean security-middleware?  I just saw today that Yu published it on npm.  I'll change to use npm instead of github.